tests/parsers/test.manifestjson.js

```diff
@@ -11,16 +11,16 @@ describe('ManifestJSONParser', function() {
     var addonLinter = new Linter({_: ['bar']});
     var manifestJSONParser = new ManifestJSONParser('blah',
                                                     addonLinter.collector);
-    assert.equal(manifestJSONParser.isValid, false);
+    expect(manifestJSONParser.isValid).toEqual(false);
     var errors = addonLinter.collector.errors;
-    assert.equal(errors.length, 1);
-    assert.equal(errors[0].code, messages.JSON_INVALID.code);
-    assert.include(errors[0].message, 'Your JSON is not valid.');
+    expect(errors.length).toEqual(1);
+    expect(errors[0].code).toEqual(messages.JSON_INVALID.code);
+    expect(errors[0].message).toContain('Your JSON is not valid.');
 
     var metadata = manifestJSONParser.getMetadata();
-    assert.equal(metadata.manifestVersion, null);
-    assert.equal(metadata.name, null);
-    assert.equal(metadata.version, null);
+    expect(metadata.manifestVersion).toEqual(null);
```

Comment:
Funny part is, this actually caught a bug that I had to fix… don't ask me why this didn't fail before though. The problem I'm talking about is:

 
https://github.com/mozilla/addons-linter/blob/master/src/parsers/manifestjson.js#L28 defines `mainfestVersion` while https://github.com/mozilla/addons-linter/blob/master/src/parsers/manifestjson.js#L177 uses `manifest_version` since that's what's supplied if a correct manifest is given. So the default value in `parsedJSON` was kinda wrong 🤔